### PR TITLE
[Android/Api] add script to build api library - @open sesame 8/19 16:47

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add script to build android-api library.
If runs this script, ext-files are downloaded and run gradle build.

TODO add repo for ext-files (or another way to get ext-files)

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: build-android-lib.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# Make directory to build NNStreamer library
+mkdir -p build_android_lib
+
+# Copy the files (native and java to build Android library) to build directory
+cp -r ./api/android/* ./build_android_lib
+
+# Get the prebuilt libraries and build-script
+pushd ./build_android_lib
+# TODO change to nnsuite repo later
+git clone https://github.com/jaeyun-jung/nnstreamer-build-ext.git
```
Comment (myungjoo) [inline build-android-lib.sh]: Git clone might be too heavy for what you need. (You don't need the history)

There are options to skip history and extract the files you need only.

Comment (jaeyun-jung) [reply build-android-lib.sh]: changed to ```svn export ...```

File: build-android-lib.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```
Comment (myungjoo) [inline build-android-lib.sh]: LGTM in overall.

However, please move this script into somewhere android-specific.

Comment (jaeyun-jung) [reply build-android-lib.sh]: moved to ```/api/android```

**Comment by jaeyun-jung** [comment]
Now updated to set nnstreamer root directory in gradle.

**Comment by jaeyun-jung** [comment]
Now updated to set nnstreamer root directory. (for the gradle build in aws)
Please review the changes again.

**Comment by jaeyun-jung** [comment]
Updated (08/12)
limit api module (api:assembleRelease)

**Pull request merged**